.github/workflows/tests_linux.yml

```diff
@@ -118,6 +118,7 @@ jobs:
         run: |
           cd main
           python -m pip install -r requirements-dev.txt
+          python -m pip install openfermionpyscf
```

Comment:
This is fine for now, let's add this somewhere that can easily be pulled in, as with the PL CI https://github.com/PennyLaneAI/pennylane/blob/4e1f7495853df620852a50ba2daa9bceeb4bd86c/.github/workflows/interface-unit-tests.yml#L427

Can we create a backlog item to improve the CI for this?

Reply:
Done. [Here](https://app.shortcut.com/xanaduai/story/48587/improve-lgpu-ci-for-adjoint-method-using-openfermionpyscf) is the story in shortcut.